linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
Ok, I can agree with that one, even though is not yet trivial in Rails, but should be shortly when Rails 5 comes out. But that should be on the JS guides, not on the linters. I think here we could have:

* JavaScript
  * ES5
  * ES6
  * Node
  * ES6 + React?

Basically if there are significant changes to the linter config, we should have a different linter for that, so we don't have to think about it when starting a new project, which is the ultimate goal.

Also, not having extra dependencies for the linter would also be good, but if can't we can at least try to minimize those. For instance, if I'm not using react, the linter should not require `eslint-plugin-react`, thus, we could have two different linter configs, one with React, one without it.

Does this make sense?